src/app/layout.tsx

```diff
@@ -22,22 +22,14 @@ export default function RootLayout({ children, modal }: { children: ReactNode; m
 	return (
 		<html lang='en'>
 			<body className={`${poppinsFont.className}`}>
-				<ComingSoon
-					className='flex h-[100vh] w-[100vw] items-center justify-center p-5 md:hidden'
-					text={
-						<span>
-							Construction zone ahead!
-							<br /> Mobile view coming soon...
-						</span>
-					}
-				/>
 				<Providers>
 					<section id='root-section'>
-						<section className='fixed h-[96vh]'>
+						<section className='hidden h-[96vh] md:fixed md:block'>
 							<AppSidebar />
 						</section>
 						<section id='main-section'>
 							<AppNavbar />
+							<AppNavMobile />
```

Comment:
we should not make another nav component for mobile, it'll just add to redundat code.
Please make it into the same component, just using styling.
For example: https://flowbite.com/docs/components/navbar/